python/cudf/cudf/io/orc.py

```diff
@@ -280,7 +280,7 @@ def read_orc(
     num_rows=None,
     use_index=True,
     timestamp_type=None,
-    use_python_file_object=True,
+    use_python_file_object=None,
```

Comment:
Please add the deprecation notice in the docstrings of these readers as well. See guidance here https://docs.rapids.ai/api/cudf/nightly/developer_guide/contributing_guide/#deprecating-and-removing-code